demos/mixer-config-quota-bookinfo.yaml

```diff
@@ -22,52 +22,53 @@ data:
         params:
       - name: default
         impl: denyChecker
-    attributes:
-      - name: source.name
-        value_type: 1 # STRING
-      - name: source.uid
-        value_type: 1 # STRING
-      - name: target.name
-        value_type: 1 # STRING
-      - name: target.uid
-        value_type: 1 # STRING
-      - name: target.service
-        value_type: 1 # STRING
-      - name: response.code
-        value_type: 2 # INT64
-      - name: response.duration
-        value_type: 10 # DURATION
-
-      # DEPRECATED ATTRIBUTES, TO BE REMOVED:
-      - name: response.latency    # Use response.duration
-        value_type: 10 # DURATION
-      - name: response.http.code  # Use response.code
-        value_type: 2 # INT64
-      - name: source.service      # Use source.name
-        value_type: 1 # STRING
-    metrics:
-      - name: request_count
-        kind: 2 # COUNTER
-        value: 2 # INT64
-        description: request count by source, target, service, and code
-        labels:
-        - name: source
+    manifests:
+      - name: istio-proxy
+        revision: "1"
+        attributes:
+        - name: source.name
```

Comment:
Not as easily as the others, there are some semantic differences. The biggest is that labels were not referred to in many different contexts, while attributes are referred to by their name across many locations; it makes sense for the attribute to "own" its name, which means we want a `message` with a `name` field instead of a `map<string, ValueType>` in the `AttributeManifest`. We can talk offline about changing that, I am a fan of making all of the configs look similar (via `map`) but I'm not sure it's semantically correct here.